http-servers/html-server.js

```diff
@@ -0,0 +1,15 @@
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+const replace = require('stream-replace');
+
+
+const server =  http.createServer((request, response) => {
+    response.writeHead(200, {'Content-Type': 'text/html'});
+    const filePath = path.join(__dirname, '/index.html');
```

Comment:
`join` method is glueing together current directory and filename into path string using slash or backslash character depending on which platform is used (linux or windows), therefore you shouldn't write any slashes or backslashes, just `path.join(__dirname, 'filename')`, that way the path string would be correct for any platform.

Reply:
Yes, Resolved.